Gemfile

```diff
@@ -7,115 +7,206 @@ end
-gem "devise", '>= 4.7.1'
+# Enforces secure passwords for users via policies.
 gem 'devise-secure_password', '~> 2.0'
-gem 'discard', '~> 1.2'
+# Sends invitations via e-mail for new users.
 gem "devise_invitable"
+# Environment variable and configuration management.
 gem "dotenv-rails"
-gem "filterrific"
+# Feature flagging and management.
 gem "flipper"
 gem "flipper-active_record"
 gem "flipper-ui"
-gem "font-awesome-rails"
-gem "font-ionicons-rails"
-gem "fullcalendar-rails"
+# Calculates latitude and longitude from an address.
 gem "geocoder"
-gem "groupdate", "~> 6.0"
-gem "image_processing"
-gem "jbuilder"
-gem "jquery-rails"
-gem "jquery-ui-rails"
+# JSON Web Token encoding / decoding (e.g. for links in e-mails)
 gem "jwt"
-gem "kaminari"
-gem "mini_racer", "~> 0.6.1"
-gem "momentjs-rails"
-gem "money-rails"
+
+##### DEPENDENCY PINS ######
+# These are gems that aren't used directly, only as dependencies for other gems.
+# Technically they don't need to be in this Gemfile at all, but we are pinning them to
+# specific versions for compatibility reasons.
+gem "mini_racer", "~> 0.6.0"
 gem "nokogiri", ">= 1.10.4"
-gem "pg", "~> 1.2.3"
-gem "simple_form"
-gem 'popper_js'
-gem "prawn-rails"
-gem "puma"
-gem "rails", "~> 6.1.4"
-gem "sass-rails"
-gem "strong_migrations", "~> 0.7.8"
-gem "sprockets", "~> 4.0.2"
-gem "toastr-rails"
-gem "uglifier", ">= 1.3.0"
-gem 'webpacker', '~> 5.4'
-gem "yajl-ruby"
-gem "recaptcha"
-gem "paper_trail" # for tracking history of InventoryItem
+gem "image_processing"
+gem "sprockets", "~> 4.0.0"
 
 group :production do
-  gem 'lograge' # Reduce the noise of logs and include custom fields to it for easier access
+  # Reduce the noise of logs and include custom fields to it for easier access
+  gem 'lograge'
+  # Profiler (third party app) showing performance and metrics.
   gem "skylight"
 end
 
 group :development, :test, :staging do
+  # Generate models based on factory definitions.
   gem 'factory_bot_rails'
+  # Ensure the database is in a clean state on every test.
   gem "database_cleaner", '1.8.5'
+  # Generate fake data for use in tests.
   gem 'faker'
 end
 
 group :development, :test do
+  # Command line tool for better object printing.
   gem "awesome_print"
+  # Scan for vulnerabilities and other static analysis.
   gem "brakeman"
-  gem "fakeredis", require: "fakeredis/rspec"
+  # Use a fake in-memory Redis instance in tests.
+  gem "fakeredis", require: "fakeredis/rspec" # NEED?
+  # Re-run specs when files change.
   gem "guard-rspec"
+  # Split tests in a suite across multiple runners.
   gem "knapsack_pro"
+  # Show core documentation in command line.
   gem "pry-doc"
+  # Rails plugin for command line.
   gem "pry-rails"
+  # Connect to a remotely running command line instance.
   gem "pry-remote"
+  # Add-on for command line to create a simple debugger.
   gem "pry-nav"
-  gem "rb-readline", "~> 0.5.3"
-  gem "rspec-rails", "~> 5.0.2"
+  gem "rb-readline", "~> 0.5.0" # NEED?
```

Comment:
Same as the prior, shouldn't we just delete? Especially given it is in development/test like the last one -- less likely to hurt anything.

Reply:
will do!